modules/contracts/src.sol/ChannelFactory.sol

```diff
@@ -44,7 +50,7 @@ contract ChannelFactory is IChannelFactory, MinimalProxyFactory {
     {
         return _calculateMinimalProxyDeploymentAddress(
             mastercopy,
-            generateSalt(alice, bob, chainId)
+            generateSalt(alice, bob)
```

Comment:
Doesn't the `chainId` still have to be included in the salt of the channel address?

Reply:
Ahh see it at the bottom, disregard